fe/fe-core/src/main/java/org/apache/doris/service/FrontendServiceImpl.java

```diff
@@ -1057,19 +1057,31 @@ private TLoadTxnBeginResult loadTxnBeginImpl(TLoadTxnBeginRequest request, Strin
         if (Strings.isNullOrEmpty(request.getLabel())) {
             throw new UserException("empty label in begin request");
         }
+        OlapTable table;
+        Database db;
         // check database
         Env env = Env.getCurrentEnv();
-        String fullDbName = ClusterNamespace.getFullName(cluster, request.getDb());
-        Database db = env.getInternalCatalog().getDbNullable(fullDbName);
-        if (db == null) {
-            String dbName = fullDbName;
-            if (Strings.isNullOrEmpty(request.getCluster())) {
-                dbName = request.getDb();
+        if (Strings.isNullOrEmpty(request.getDb()) && request.getTableId() != -1) {
```

Comment:
IfSetTableID && getTableID()
```suggestion
        if (Strings.isNullOrEmpty(request.getDb()) && request.getTableId() != -1) {
```